drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableGraphEditorPresenter.java

```diff
@@ -108,28 +109,41 @@
 
     protected ObservablePath.OnConcurrentUpdateEvent concurrentUpdateSessionInfo = null;
     protected Access access = new Access();
+    protected Integer originalGraphHash;
 
     private class LoadGraphLatch {
 
         private int dtGraphElementCount;
-        private Path dtToSelectPath;
+        private Command onAllDocumentGraphEntriesLoadedCommand;
+        private ParameterizedCommand<GuidedDecisionTableView.Presenter> onDocumentGraphEntryLoadedCommand;
 
         private LoadGraphLatch( final int dtGraphElementCount,
-                                final Path dtToSelectPath ) {
+                                final ParameterizedCommand<GuidedDecisionTableView.Presenter> onDocumentGraphEntryLoadedCommand ) {
+            this( dtGraphElementCount,
+                  onDocumentGraphEntryLoadedCommand,
+                  () -> {/*Do nothing*/} );
+        }
+
+        private LoadGraphLatch( final int dtGraphElementCount,
+                                final ParameterizedCommand<GuidedDecisionTableView.Presenter> onDocumentGraphEntryLoadedCommand,
+                                final Command onAllDocumentGraphEntriesLoadedCommand ) {
             this.dtGraphElementCount = dtGraphElementCount;
-            this.dtToSelectPath = dtToSelectPath;
+            this.onDocumentGraphEntryLoadedCommand = onDocumentGraphEntryLoadedCommand;
+            this.onAllDocumentGraphEntriesLoadedCommand = onAllDocumentGraphEntriesLoadedCommand;
         }
 
-        private void fireDecisionTableSelectedEvent( final GuidedDecisionTableView.Presenter dtPresenter ) {
```

Comment:
Hello @manstis . Thanks for your fix in such short time. I have just short question. I see you deleted the method `fireDecisionTableSelectedEvent`, however you added the same code twice in your PR.

https://github.com/manstis/drools-wb/blob/3a5cf506f20a302a819d246c173d87a0cbbf9fee/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableGraphEditorPresenter.java#L446

https://github.com/manstis/drools-wb/blob/3a5cf506f20a302a819d246c173d87a0cbbf9fee/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableGraphEditorPresenter.java#L574

Is it intended? Couldn't be worth to keep the private method? Since there are two occurrences of the same code.

What do you think?